src/cli-main.ts

```diff
@@ -253,6 +254,66 @@ program
     cliUtils.logAndExit(0, copywrite);
   });
 
+program
+  .command("create-auth-token <secretKey>")
+  .description("Generates an authentication token that can be used to get detailed EdgeWorker debug response headers.  \
+  The secret key (hex-digit based, min 64 chars) that is configured for the Akamai property in which the EdgeWorker executes.")
+  .alias("auth")
+  .option("--acl <aclPath>", "The path prefix of the response pages which require debugging; this value is used to restrict for which pages the token is valid. \
+  The default value if not specified is \"/*\". This option is mutually exclusive to the --url option; only use one or the other.")
+  .option("--url <urlPath>", "The exact path (including filename and extension) of the response page which requires debugging; this value is used as a salt for \
+  generating the token, and the URL does NOT appear in the final token itself. The generated token is only valid for the exact URL. This option is mutually \
+  exclusive to the --acl option; only use one or the other.")
+  .option("--expiry <expiry>", "The number of minutes during which the token is valid, after which it expires. Max value is 60 minutes; default value is 15 minutes.")
+  .action(async function (secretKey, options) {
+
+    if(!secretKey) {
+      cliUtils.logAndExit(1, "ERROR: The secret key specified in the property in which the EdgeWorker executes must be supplied in order to generate an authentication token.");
+    } else if(secretKey.length < 64) {
+      cliUtils.logAndExit(1, "ERROR: The secret key specified in the property in which the EdgeWorker executes must have at least 64 characters.");
+    } else if(secretKey.length % 2 != 0) {
+      cliUtils.logAndExit(1, "ERROR: The secret key specified in the property in which the EdgeWorker executes must have an even number of hex characters.");
+    } else if(!secretKey.match(/^[0-9a-fA-F]+$/)) {
+      cliUtils.logAndExit(1, "ERROR: The secret key specified in the property in which the EdgeWorker executes must contain only hex characters: [0-9a-f]");
+    } else {
+      secretKey = secretKey.toLowerCase();
+    }
+
+    var expiry = 15; // Use 15 minutes as the default value
+    if(options.expiry) {
+      expiry = parseInt(options.expiry);
+      if(isNaN(expiry)) {
+        cliUtils.logAndExit(1, "ERROR: The expiry is invalid. It must be an integer value (in minutes) representing the duration of the validity of the token.");
+      } else if(expiry < 1 || expiry > 60) {
+        cliUtils.logAndExit(1, "ERROR: The expiry is invalid. It must be an integer value (in minutes) between 1 and 60.");
+      }
+    }
+
+    var path = "/*";
+    var isACL = true;
+    if(options.acl) {
+      if(options.url) {
+        cliUtils.logAndExit(1, "ERROR: The --acl and --url parameters are mutually exclusive; please use only one parameter. Specifying neither will result in a \
```

Comment:
Formatting: When I hit this code path on my MacOS terminal this message is formatted with about 9 spaces between "a" and "default":
![Screen Shot 2020-09-24 at 8 43 01 PM](https://user-images.githubusercontent.com/13912416/94224204-a38a5680-fea6-11ea-807e-be18039ee4d1.jpg)

Reply:
Yes, I can see the same thing. I will fix and update.